### PR TITLE
pkg/table,pkg/tablecodec: add `OWNERS` files

### LINKED ISSUE #47854: add `OWNERS` file for `pkg/table` folder

### PR DESCRIPTION
**Pull request by wuhuizuo**
### What problem does this PR solve?

Issue Number: close #47854, #47855

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [x] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -44,3 +44,12 @@ aliases:
     - lance6716
     - tangenta
     - wjhuang2016
+  sig-approvers-table: # approvers for table packages.
+    - Benjamin2037
+    - D3Hunter
+    - gmhdbjd
+    - lance6716
```
Comment (lance6716) [inline OWNERS_ALIASES:51]:
```suggestion
```

in fact I'm not familiar with this package

Comment (wuhuizuo) [reply OWNERS_ALIASES:51]: OK.

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -44,3 +44,11 @@ aliases:
     - lance6716
     - tangenta
     - wjhuang2016
+  sig-approvers-table: # approvers for table packages.
```
Comment (cfzjywxk) [inline OWNERS_ALIASES:47]: For the OLTP read write path, please add me.

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -44,3 +44,11 @@ aliases:
...
+    - Benjamin2037
+    - D3Hunter
+    - gmhdbjd
+    - tangenta
+    - wjhuang2016
+    - ywqzzy
+    - zimulala
```
Comment (wuhuizuo) [inline OWNERS_ALIASES:54]:
```suggestion    
    - Benjamin2037
    - cfzjywxk
    - D3Hunter
    - gmhdbjd
    - tangenta
    - wjhuang2016
    - ywqzzy
    - zimulala
```

File: OWNERS_ALIASES (hunk 2 above)
Comment (wuhuizuo) [reply OWNERS_ALIASES:47]: done

File: OWNERS_ALIASES [hunk 4]
```diff
@@ -51,4 +51,13 @@ aliases:
     - lance6716
     - tangenta
     - wjhuang2016
-    - ywqzzy
\ No newline at end of file
+    - ywqzzy
+  sig-approvers-table: # approvers for table packages.
+    - Benjamin2037
+    - cfzjywxk
+    - D3Hunter
```
Comment (D3Hunter) [inline OWNERS_ALIASES:58]:
```suggestion
```
not familier with this pkg

**Review by tangenta — APPROVED** [review]

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**